frontend/src/components/common/header/navbar.jsx

```diff
@@ -69,23 +70,33 @@ class NavBar extends Component {
             <Navbar collapseOnSelect expand="lg">
               <Navbar.Toggle className="ml-3" onClick={this.toggle} />
               <Navbar.Collapse isOpen={this.state.isOpen} navbar>
-                <Nav className="mx-auto my-auto align-links">
+                <Nav className="mx-auto">
                   <NavDropdown
                     className="nav-link"
                     to="/resources"
                     title="Youth Resources"
                     id="nav-dropdown"
                   >
-                    <NavDropdown.Item eventKey="4.1">
-                      Hospitals
+                    <NavDropdown.Item>
+                      <HashLink eventKey="4.1" to="/resources#hospital">
+                        Hospitals
+                      </HashLink>
                     </NavDropdown.Item>
-                    <NavDropdown.Item eventKey="4.2">
-                      Urgent care
+                    <NavDropdown.Item>
+                      <HashLink eventKey="4.2" to="/resources#uc">
+                        Urgent Care
+                      </HashLink>
                     </NavDropdown.Item>
-                    <NavDropdown.Item eventKey="4.3">
-                      Emergency Resources
+                    <NavDropdown.Item>
+                      <HashLink eventKey="4.3" to="/resources#shelter">
```

Comment:
If assigning the ID for Hashlink can you stylize the CSS on the ID instead of adding another class?